### PR TITLE
Update color format caps and validate the blending

### PR DESCRIPTION
**Pull request by kvark**
Follow-up to  #1118

***
<a href="https://pr-preview.s3.amazonaws.com/kvark/gpuweb/pull/1132.html" title="Last updated on Oct 6, 2020, 1:19 PM UTC (99a849a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/gpuweb/gpuweb/1132/cc5dde3...kvark:99a849a.html" title="Last updated on Oct 6, 2020, 1:19 PM UTC (99a849a)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -3862,6 +3849,21 @@ enum GPUBlendOperation {
...
+        - {{GPUColorStateDescriptor}} |descriptor|
+
+    Return `true`, if and only if, all of the following conditions are satisfied:
+
+        1. |descriptor|.{{GPUColorStateDescriptor/format}} is listed in {#plain-color-formats}
+            with {{GPUTextureUsage/OUTPUT_ATTACHMENT}} capability.
+        1. |descriptor|.{{GPUColorStateDescriptor/alphaBlend}} and |descriptor|.{{GPUColorStateDescriptor/colorBlend}}
+            are either both defaults, or |descriptor|.{{GPUColorStateDescriptor/format}} has
+            {{GPUTextureComponentType/"float"}} component type.
+        1. |descriptor|.{{GPUColorStateDescriptor/writeMask}} is less than 16
```
Comment (kainino0x) [inline spec/index.bs:3864]:
```suggestion
        1. |descriptor|.{{GPUColorStateDescriptor/writeMask}} is less than 16.
```

File: spec/index.bs [hunk 2]
```diff
@@ -7305,49 +7293,49 @@ All packed texture formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsa
             <th>Extension
     </thead>
     <tr>
-        <td data-componenttype=float>{{GPUTextureFormat/rgb9e5ufloat}}
+        <td>{{GPUTextureFormat/rgb9e5ufloat}}
```
Comment (kainino0x) [inline spec/index.bs:7296]: I assume we should add a component type column to this table?

File: spec/index.bs [hunk 3]
```diff
@@ -7014,214 +7002,214 @@ The "Filter" column specifies whether textures of this format can be sampled in
     </thead>
```
Comment (kainino0x) [inline spec/index.bs:7002]: nit on unchanged line above this: "Component Types" should be "Component Type" I think

**Review by kainino0x — APPROVED** [review]

File: spec/index.bs (hunk 2 above)
Comment (Kangz) [reply spec/index.bs:7296]: There is already one as the third column.

File: spec/index.bs [hunk 4]
```diff
@@ -6999,7 +6987,7 @@ All plain color formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/
 
 Only formats with {{GPUTextureComponentType}} {{GPUTextureComponentType/"float"}} can be blended.
 
-The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}} usage in the core API, including both {{GPUBindingType/"readonly-storage-texture"}} and {{GPUBindingType/"writeonly-storage-texture"}}. If there is a &star; mark, the atomic operations on the storage are also supported for this format.
+The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}} usage in the core API, including both {{GPUBindingType/"readonly-storage-texture"}} and {{GPUBindingType/"writeonly-storage-texture"}}.
```
Comment (Kangz) [inline spec/index.bs:6990]: Out of curiosity, why was the notion of atomic texture ops removed?

**Review by Kangz — APPROVED** [review]

File: spec/index.bs (hunk 4 above)
Comment (kvark) [reply spec/index.bs:6990]: Because MSL doesn't have this even on the type level :(

**Pull request merged**